### DISCUSSION TITLE
Did the view clip URL change?

### DISCUSSION THREAD

**Discussion by BNWEIN** [blakeblackshear/frigate#15143] [General Support]
I am using a HA automation to send notifications to my mobile device:l. It seems since moving to version 15 my view clip button on my notification no longer works. The URL it's going to is: 

https://my.domain.co.uk/api/frigate/notifications/1732355002.725165-zrabdp/side_cam/clip.mp4

When I go there, I am getting ![image](https://github.com/user-attachments/assets/a3cb90c1-fc6e-49d4-8e0e-152c8b9f822d)

Any help appreciated!

**Comment by NickM-27** [comment]
This is a home assistant bug
